src/animators/animator.ts

```diff
@@ -1,5 +1,39 @@
 module Plottable {
+export module Animators {
+  export type d3SelectionOrTransition = d3.Selection<any>|d3.Transition<any>;
 
+  export type EasingFunction = (t: number) => number;
+  export type EasingFunctionSpecifier = string|EasingFunction;
```

Comment:
Instead of `Plottable` needing to export / declare these types, would it make more sense to submit a pull request to DefinitelyTyped for D3 to expose this?